src/EditorFeatures/Core/Shared/Tagging/TagSources/ProducerPopulatedTagSource.cs

```diff
@@ -119,7 +118,6 @@ internal abstract partial class ProducerPopulatedTagSource<TTag> : TagSource<TTa
             _cachedTags = ImmutableDictionary.Create<ITextBuffer, TagSpanIntervalTree<TTag>>();
 
             _eventSource = eventSource;
-            _bufferToRelatedTagSource = bufferToRelatedTagSource;
```

Comment:
This was the unused field.  Because it was unused i could remove the corresponding parameter in the constructor.  I then just walked backward removing the instances from the callsites.